tests/unit/modules/test_inspect_collector.py

```diff
@@ -33,7 +34,23 @@
 from salt.modules.inspectlib.collector import Inspector
 
 
+HAS_SYMLINKS = None
+
+
+def no_symlinks():
```

Comment:
There may be a more 'salty' way of doing this. I looked around a bit but didn't see one.

Reply:
There's also an option in the git installer for Windows where you have to enable symlinks on Windows. I don't know how you would check for that.

Reply:
I'm sure that option will just set the global config option for symlinks. This check will encompass that.